lib/mongo/cluster.rb

```diff
@@ -118,6 +125,34 @@ def initialize(seeds, monitoring, options = Options::Redacted.new)
 
       @connecting = false
       @connected = true
+
+      if options[:scan] != false
```

Comment:
I'm a little confused about this diff; it looks like this is only adding things in the case that the option is true; wouldn't that mean that the option being false would lead to the same behavior as before (which is still doing I/O)?

Reply:
Oh, never mind, I missed the deletion of `monitor.scan!` below